includes/sanitizers/class-amp-gallery-block-sanitizer.php

```diff
@@ -140,8 +140,43 @@ public function sanitize() {
 					'layout' => 'responsive',
 				]
 			);
+
 			foreach ( $images as $image ) {
-				$amp_carousel->appendChild( $image );
+				$slide = AMP_DOM_Utils::create_node(
+					$this->dom,
+					'div',
+					[ 'class' => 'slide' ]
+				);
+
+				// Ensure the image fills the entire <amp-carousel>, so the possible caption looks right.
+				if ( 'amp-img' === $image->tagName ) {
+					$image->setAttribute( 'layout', 'fill' );
```

Comment:
`layout="fill"` appears a lot in the documentation for [Image Galleries with amp-carousel](https://amp.dev/documentation/examples/multimedia-animations/image_galleries_with_amp-carousel/?format=websites)